_posts/2023-05-04-dedekker/2023-05-04-dedekker-en.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Lecture: The 1897-99 Belgica Expedition"
+layout: 'post'
+lang: en
+ref: ubb75years
+banner: /assets/images/May23/web_banner.jpg
+---
+
+![](/assets/images/May23/web_banner.jpg){: .align-center}
+
+## The 1897-99 Belgica Expedition that first overwintered in Antarctic waters
+_A tale of extraordinary scientific achievements and of human endurance_
+
+Patrick De Deckker
+Emeritus Professor, The Australian National University
+
+
+The Emil G. Racoviță Institute for the Study of Life in Extreme Conditions and the Emil G. Racoviță Foundation for Science and Exploration in Extreme Environments invites you to a lecture by Professor emeritus Patrick De Deckker, Australian National University and Australian Academy of Science, entitled _The 1897-99 Belgica expedition that first overwintered in Antarctic waters - A tale of extraordinary scientific achievements and of human endurance_.
```

Comment:
"The 1897-99 Belgica expedition that first overwintered in Antarctic waters - A tale of extraordinary scientific achievements and of human endurance".

Reply:
De fapt cum crezi că e mai ok în EN, numai italic sau între ghilimele.

Reply:
L-am split in titlu și sub-titlu. După mine un titlu trebuie sa aibă mai putin de 10 cuvinte :)

Atașate sunt cele snapshots pentru cele 2 versiuni.
<img width="1304" alt="Screenshot 2023-04-26 at 20 32 47" src="https://user-images.githubusercontent.com/3770874/234518296-c4397468-2d07-44b5-b4e0-d5aca3dc1153.png">
<img width="1214" alt="Screenshot 2023-04-26 at 20 33 32" src="https://user-images.githubusercontent.com/3770874/234518329-3c51a8f0-6ace-4284-8fab-dca09ed9932e.png">

Reply:
Ok :), lasă varianta cu split titlu și subtitlu. De acord cu numărul de cuvinte în titlu.